v1/src/test/java/com/google/cloud/teleport/spanner/ddl/DdlTest.java

```diff
@@ -864,6 +865,25 @@ public void pgSequences() {
     assertNotNull(ddl.hashCode());
   }
 
+  @Test
+  public void protobundle() {
+    Ddl.Builder builder = Ddl.builder();
+    builder.mergeProtoBundle(
+                ImmutableSet.of(
+                    "com.google.cloud.teleport.spanner.tests.TestMessage",
+                    "com.google.cloud.teleport.spanner.tests.TestEnum"));
+    Ddl ddl = builder.build();
+    String expectedProtoBundle = "CREATE PROTO BUNDLE ("
+        + "\n\tcom.google.cloud.teleport.spanner.tests.TestMessage,"
+        + "\n\tcom.google.cloud.teleport.spanner.tests.TestEnum\n)";
```

Comment:
```suggestion
        + "com.google.cloud.teleport.spanner.tests.TestMessage,"
        + "com.google.cloud.teleport.spanner.tests.TestEnum)";
```

Since whitespace is removed before comparision

Reply:
The helper method doesn't compress \n and \t as whitespace characters. So, keeping as is.